Buildings/UsersGuide/ReleaseNotes/Version_11_0_1.mo

```diff
@@ -17,6 +17,14 @@ The following <b style=\"color:blue\">new libraries</b> have been added:
     <td valign=\"top\">xxx.
     </td>
     </tr>
+<tr><td valign=\"top\">Buildings.Fluid.CHPs.OrganicRankine
+    </td>
+    <td valign=\"top\">Package of an organic Rankine cycle,
+                       including working fluid data records and district system integration.
```

Comment:
I think the "district system integration" can be removed, or better explain what this package does regarding the district system integration. In my view it is just a component, how users will use it should not be here.

Reply:
Removed this wording in 23b86db44167222602ab60ee874a2ea1a8d166de.